packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -1,77 +1,77 @@
-import React from "react"
+import { CoreColumn, EntityName, OwidTable } from "@ourworldindata/core-table"
```

Comment:
My editor autosorted the imports into alphabetical order. I can revert that if it is confusing.

Reply:
I don't mind it :)